Client/Frontend/Browser/SearchEngines.swift

```diff
@@ -118,13 +118,36 @@ class SearchEngines {
         disabledEngineNames[engine.shortName] = true
     }
 
+    func reloadEngines() {
+        self.orderedEngines = getOrderedEngines()
+    }
+
+    func deleteEngine(engine: OpenSearchEngine) {
```

Comment:
I think this is related to the change I made with the engineId. I'll look into this.